calico-cloud_versioned_docs/version-3.15/reference/component-resources/node/configuration.mdx

```diff
@@ -300,8 +300,7 @@ Substitute `[flag]` with one or more of the following.
 - `-felix-ready`
 
 The BIRD readiness endpoint ensures that the BGP mesh is healthy by verifying that all BGP peers are established and
-no graceful restart is in progress. If the BIRD readiness check is failing due to unreachable peers that are no longer
```

Comment:
I think we shouldn't be making changes to versions we're no longer using or serving. In other words I don't think you should make this change to the CC 3.15 docs.